docs/troubleshooting.md

```diff
@@ -56,19 +56,25 @@ You may also find the StackOverflow topics relevant
 
 ## Bugs and Feature requests
 
-If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system](https://github.com/GoogleCloudPlatform/kubernetes/issues).
+If you have what looks like a bug, or you would like to make a feature request, please use the [Github issue tracking system](https://github.com/kubernetes/kubernetes/issues).
 
 Before you file an issue, please search existing issues to see if your issue is already covered.
 
+If filing a bug, please include information about how to reproduce the problem.
+
 # Help! My question isn't covered!  I need help now!
 
+## Stackoverflow
```

Comment:
I would suggest re-ordering this troubleshooting guide to direct new users first to Stackoverflow for troubleshooting help, and only after they have exhausted that option, consider filing bug reports or feature requests.  From my experience things that look like bugs or missing features to new users often evaporate once more experienced users offer alternative approaches or configuration debugging support.